bin/automerge-prs

```diff
@@ -1,39 +1,65 @@
 #!/usr/bin/env bash
 set -euo pipefail
-
 shopt -s nullglob
 
-if [[ -z "${GITHUB_TOKEN:-""}" ]]; then
-  echo "GITHUB_TOKEN not set" >&2
-  exit 1
-fi
-
-if (($# < 3)); then
-  echo "Usage: automerge-prs <owner/repo> <quarantine-days> <strategy> [exclude-title-regex]" >&2
-  exit 64
-fi
-
-repo=$1
-quarantine_days=$2
-strategy=$3
-exclude_title_regex=${4:-""}
-
-if [[ ! "$strategy" =~ merge|rebase|squash ]]; then
+if [[ ! "$STRATEGY" =~ merge|rebase|squash ]]; then
   echo "Invalid strategy: must be merge, rebase, or squash" >&2
   exit 64
 fi
 
 : "${DRY_RUN:=0}"
 
+: "${GH_ACTOR:=""}"
+: "${GH_EVENT:=""}"
+: "${GH_PR_ACTION:=""}"
+: "${GH_PR_NUMBER:=""}"
+: "${GH_PR_TITLE:=""}"
+
+is_dependabot() {
+  [[ "$GH_ACTOR" == 'dependabot[bot]' ]] && [[ "$GH_EVENT" == 'pull_request' ]]
+}
+
+exclude_by_title() {
+  [[ -n "$EXCLUDE_TITLE_REGEX" ]] && [[ "$1" =~ $EXCLUDE_TITLE_REGEX ]]
+}
+
 gh_pr() {
-  gh pr --repo "$repo" "$@"
+  gh pr --repo "$GH_REPO" "$@"
 }
 
 tmp=$(mktemp -d)
 trap 'rm -rf "$tmp"' EXIT
 
+if is_dependabot && ! exclude_by_title "$GH_PR_TITLE"; then
+  now_s=$(date +"%s")
+  until_s=$((now_s + (QUARANTINE_DAYS * 24 * 60 * 60)))
+  until=$(date -d "@$until_s" +"%Y-%m-%d")
+
+  case "$GH_PR_ACTION" in
+    opened)
+      cat >"$tmp/message" <<EOM
+:heavy_check_mark: If all status checks pass, and no other reviews are submitted, [mergeabot][] will merge this PR after $QUARANTINE_DAYS day(s), on $until.
+
+As long as that's OK, no other action is necessary.
+
+[mergeabot]: https://github.com/freckle/mergeabot-action
+EOM
```

Comment:
You'll have to imagine this comment happened. It was broken when this PR opened. The `synchronized` comment proves it works now, but it looks a little confusing without the additional context of this comment appearing above.